test/e2e/test_kfservice_creating.py

```diff
@@ -0,0 +1,43 @@
+import time
```

Comment:
We can rename the file to be `test_tensorflow_inference_service.py`, we need tests for each model framework and each one has predictor/transformer/explainer test cases.

Reply:
Done